components/navigation/navigation.css

```diff
@@ -0,0 +1,37 @@
+.navigation{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 10px;
+    background-color: var(--white);
+}
+
+.search-text{
+    flex-grow: 1;
+    padding: 0.5rem;
+    border: 1px solid var(--default-color);
+    border-radius: 10px;
+    margin: 0%;
+    margin-right: 10px;
+}
+
+.navigation a{
+    font-weight: 600;
+    color : var(--default-color);
+    text-decoration: none;
+    margin-right: 10px;
+    
+}
+
+.navigation a:hover{
+    color: var(--primary-color);
```

Comment:
great work adding color by variables! Makes code clean.

Reply:
@abhishek-bhatkar Thank You!